lib/customs.js

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var P = require('./promise')
+var Pool = require('./pool')
+
+function Customs() {
+  this.pool = new Pool("http://127.0.0.1:7000", { timeout: 1000 })
+}
+
+Customs.prototype.checkAuthenticated = function (ip, token, uid) {
+  return this.pool.post(
+      '/checkAuthenticated',
+      {
+        ip: ip,
+        token: token,
+        uid: uid
+      }
+  ) 
```

Comment:
Yes please, and also note it in the API docs for the relevant endpoints, as done at e.g. https://github.com/mozilla/fxa-auth-server/blob/master/docs/api.md#response-2